ITC205_BaseLine_Code/src/Main.java

```diff
@@ -1,5 +1,7 @@
 import java.text.SimpleDateFormat;
 import java.util.Scanner;
+//Ready for static review
```

Comment:
D19 – fix line wrap on block of code lines 15-19

Reply:
Fixed lines 18-21 (lines slightly out due to adding /comment line in top of class)

Reply:
Confirmed Fixed - Facilitator